src/index.tsx

```diff
@@ -214,6 +216,20 @@ const MuiTelInput = React.forwardRef(
             {...MenuProps}
           />
         ) : null}
+        {!disableDropdown && allowSearch ? (
+          <FlagsAutocomplete
```

Comment:
too much duplications.. Would be nice to keep everything less verbose, and having only a logic for the visibility of the autocomplete (visible or not visible).